docs/conf.py

```diff
@@ -1,300 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I see that we are getting rid of all sphinx infrastructure, and switching to markdown. Reasonable.

Reply:
Yeah, If we end up in a situation where we expand on these docs and put them up somewhere then we can definitely revisit adding it back in. But for now it's just cruft as it wasn't being used.